File: api/src/main/java/com/redbee/io/domain/Restaurant.java
package com.redbee.io.domain;

import java.util.List;

/**
 * Created by channo on 26/08/16.
 */
public class Restaurant {

    private String id;
    private String name;
    private List<Dish> dishes;

    public String getId() {
        return id;
    }

    public List<Dish> getDishes() {
        return dishes;
    }

    public void setDishes(List<Dish> dishes) {
        this.dishes = dishes;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }



}

File: hackatom-ui/hackatom-ui/src/app/domain/restaurant.ts
import { Dish } from './dish'

export class Restaurant {
	id: string;
	name: string;
	dishes: Array<Dish>;

	constructor(id:string, name:string, dishes: Dish[]) {
		this.id = id;
		this.name = name;
		this.dishes = dishes;
	}
}
